components/RepositoryList.tsx

```diff
@@ -18,10 +20,49 @@ const Loader = () => (
   </div>
 );
 
-export const RepositoryList = ({ repositories }: RepositoryListProps) => {
+export const RepositoryList = ({ repositories, dtOrder, starOrder  }: RepositoryListProps) => {
   const itemsPerScroll = 15;
   const [items, setItems] = useState(itemsPerScroll);
 
+  if (dtOrder){
+    repositories.sort(function (a, b) {
+      if (a.last_modified < b.last_modified) {
```

Comment:
I haven't checked, but in many languages `repositories.sort((a, b) => a.last_modified < b.last_modified)` should be enough for sorting, no need to explicitly write all the cases. Same for `created_at` below.